tuf/repository_tool.py

```diff
@@ -1815,8 +1814,8 @@ def add_restricted_paths(self, restricted_paths, child_rolename):
       # Append a trailing path separator with os.path.join(path, '').
       targets_directory = os.path.join(self._targets_directory, '')
       if not restricted_path.startswith(targets_directory):
-        raise securesystemslib.exceptions.Error(repr(restricted_path) + ' does'
-          ' not live under the repository\'s targets'
+        logger.debug(repr(restricted_path) + ' does not live under the'
+            ' repository\'s targets'
```

Comment:
Nitpick: I would indent both continuing lines consistently and maybe fill up the second line.